sdk/maps/azure-maps-search/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2024-12-12)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Initial release
```

Comment:
It shouldn't be "Initial release". Please use:
* Support Search API `2023-06-01`
and add a new section `Breaking Changes` saying remove Search API 1.0